bounded-collections/src/lib.rs

```diff
@@ -16,13 +16,15 @@ pub extern crate alloc;
 pub mod bounded_btree_map;
 pub mod bounded_btree_set;
 pub mod bounded_vec;
+mod const_int;
```

Comment:
There isn't a particular reason, it's just that other ConstU* types are included in the top-level module, but since my implementation has many macros or helper types, I wrote them in a separate module solely for maintainability and readability purposes. I set them as private for that reason. If it would be better to expose them as public, I can do so.